src/invidious/data_structs/invidious/video_preferences.cr

```diff
@@ -0,0 +1,29 @@
+module InvidiousStructs
+  # Struct containing all values for video preferences
+  struct VideoPreferences
```

Comment:
Can you please revert your changes on this struct? I'm already working on a preferences overhaul :)
Same for `Config` and `Preferences`.